File: code/coding_pratice_Interview/Amstrong_number/Amstrong_number.js
const Amstrong_number = (n) =>{
    let r, n1, sum = 0;
    n1 = n;
    i = 3
    while(n > 0){
        r = n % 10;
        sum = sum + r*r*r*r
        n = Math.trunc(n/10)
    }
    if(n1 == sum){
        console.log("Amstrong number");
    }else{
        console.log("Not a Amstrong number");
    }
}

// Amstrong_number(8208)

File: code/coding_pratice_Interview/Amstrong_number/Amstrong_number.py
def Amstrong_number(num):
    sum = 0
    n1 = num
    for i in num:
        r = num % 10
        sum = sum + r * r * r 
        num = num/10
    if(n1 == sum):
        print("Amstrong number")
    else:
        print("Not a Amstrong Number")    

Amstrong_number(2)